packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -55,9 +66,32 @@ export default function BookmarkFeedLayout({
       <FeedPageHeader className="mb-5">
         <h3 className="font-bold typo-callout">Bookmarks</h3>
       </FeedPageHeader>
-      <CustomFeedHeader className="relative mb-6">
+      <CustomFeedHeader className="flex relative mb-6">
         {searchChildren}
+
+        <Button
+          className="ml-4 btn-secondary hidden laptop:flex"
+          buttonSize="medium"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+        >
+          Share bookmarks
+        </Button>
+        <Button
+          className="ml-4 btn-secondary block laptop:hidden"
+          buttonSize="medium"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+          iconOnly
```

Comment:
This prop shouldn't be necessary anymore - it is not utilized in the `Button` component itself as there's a specific variable that handles it inside.
```suggestion
```